src/plugins/workspace/public/components/workspace_form/workspace_permission_setting_panel.tsx

```diff
@@ -96,31 +113,36 @@ const UserOrGroupSection = ({
         )
       );
     },
-    [onChange, permissionSettings]
+    [handlePermissionSettingsChange, permissionSettings]
+  );
+
+  const handleTypeChange = useCallback<WorkspacePermissionSettingInputProps['onTypeChange']>(
+    (type, index) => {
+      handlePermissionSettingsChange?.(
+        permissionSettings.map((item, itemIndex) =>
+          index === itemIndex ? { id: item.id, type, modes: item.modes } : item
+        )
+      );
+    },
+    [handlePermissionSettingsChange, permissionSettings]
   );
 
   return (
     <div>
       <EuiFlexGroup gutterSize="m">
         <EuiFlexItem style={{ maxWidth: 400 }}>
           <EuiCompressedFormRow
-            label={
-              type === WorkspacePermissionItemType.User
-                ? i18n.translate('workspaceForm.permissionSetting.userLabel', {
-                    defaultMessage: 'User',
-                  })
-                : i18n.translate('workspaceForm.permissionSetting.groupLabel', {
-                    defaultMessage: 'User group',
-                  })
-            }
+            label={i18n.translate('workspaceForm.permissionSetting.collaboratorLabel', {
+              defaultMessage: 'Collaborator',
+            })}
           >
             <></>
```

Comment:
Can we remove this empty children element?

Reply:
sorry I forgot to delete it